sympy/stats/tests/test_stochastic_process.py

```diff
@@ -108,6 +112,29 @@ def test_DiscreteMarkovChain():
     assert variance(X[1], Eq(X[0], 1)) == Rational(8, 9)
     raises(ValueError, lambda: E(X[1], Eq(X[2], 1)))
 
+
+def test_sample_stochastic():
```

Comment:
Are these tests deterministic? I cannot find the value of seed being fixed for these tests in the diff.

Reply:
They are random! Do you mean using sampling seeds to make them produce same result always?

Reply:
Yes, tests should be deterministic. They can be done adding a line like,
```suggestion
def test_sample_stochastic():
    random.seed(0)
```